package.json

```diff
@@ -10,15 +10,15 @@
     "access": "public"
   },
   "sideEffects": false,
-  "main": "dist/index.js",
+  "main": "dist/index.cjs.js",
   "module": "dist/index.js",
   "types": "dist/index.d.ts",
```

Comment:
pretty sure I've broken this and need to figure out how to preserve the definition files using rollup's typescript plugin